FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -0,0 +1,313 @@
+            }
+            return allAssets;
+        }
+    }
+    /**
+     * This is used to return all the distinct statuses in a project/workspace (at the moment workspaceID)
+     * @param dbClient - A client for connection to the DB
+     * @param workspaceId - The ID of the wanted workspace
+     * @return a list of strings holding the different statuses in a project/workspace
+     */
+    public List<String> getStatusList(DatabaseClient dbClient, String workspaceId) {
+        List<String> statusList = new ArrayList<>();
+        Statement statement =
+                Statement.newBuilder(
+                        "SELECT DISTINCT status FROM Main_Assets "
+                                + "WHERE workspaceId = @workspaceId")
+                        .bind("workspaceId")
+                        .to(workspaceId)
+                        .build();
+        try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+            while (resultSet.next()) {
+                if (!resultSet.isNull("status")) {
+                    statusList.add(resultSet.getString("status"));
+                }
+            }
+        }
+        return statusList;
+    }
+    /**
+     * This is used to return all the distinct locations in a project/workspace (at the moment workspaceID)
+     * @param dbClient - A client for connection to the DB
+     * @param workspaceId - The ID of the wanted workspace
+     * @return a list of strings holding the different locations in a project/workspace
+     */
+    public List<String> getLocationList(DatabaseClient dbClient, String workspaceId) {
+        List<String> locationList = new ArrayList<>();
+        Statement statement =
+                Statement.newBuilder(
+                        "SELECT DISTINCT location FROM Main_Assets "
+                        + "WHERE workspaceId = @workspaceId")
+                        .bind("workspaceId")
+                .to(workspaceId)
+                .build();
+        try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+            while (resultSet.next()) {
+                if (!resultSet.isNull("location")) {
+                    locationList.add(resultSet.getString("location"));
+                }
+            }
+        }
+        return locationList;
+    }
+    /**
+     * This is used to execute a query that was built dynamically and column types are unknown and return a list of data
+     * @param statement - A statement that holds the query
+     * @param dbClient - A client for connection to the DB
+     * @param columnNames - A list of column names to use for getting real types
+     * @param columnTypes - A list of column types to use with Arrays
+     * @return a list os lists where each row holds information of one asset in strings (to use in template)
+     */
+    public List<List<String>> executeQueryAndReturnList(Statement statement, DatabaseClient dbClient, List<String> columnNames,  List<String> columnTypes){
```

Comment:
Do you have access to a formatter that will wrap these long lines?
(The Google style guides limits lines to 100 cols)

Also indenting by 2 cols makes deep nesting easier to read.